doc/cephfs/client-auth.rst

```diff
@@ -146,3 +146,23 @@ The optional ``{network/prefix}`` is a standard network name and
 prefix length in CIDR notation (e.g., ``10.3.0.0/16``).  If present,
 the use of this capability is restricted to clients connecting from
 this network.
+
+Filesystem information restriction
+==================================
+
+If desired, the monitor cluster can present a limited view of the filesystems
+available. In this case, the monitor cluster will only inform clients about
+filesystems specified by the administrator. Other filesystems will not be
+reported and commands affecting them will be ignored as if the filesystems
+did not exist.
+
+::
+
+ client.0
+     key: AQAz7EVWygILFRAAdIcuJ12opU/JKyfFmxhuaw==
+     caps: [mon] allow r, allow fsid=1, allow fsid=2
```

Comment:
ditto.  the first allow r applies to all fsids.  the fsid= has to be a property of each allow to restrict it.